calibration/DOFSTA/ATAPL-58694-00002__20170909.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+43-2484,CC_oxygen_signal_slope,0.4563,<Soc>
+43-2484,CC_residual_temperature_correction_factor_a,-3.9245e-003,<A>
+43-2484,CC_residual_temperature_correction_factor_b,1.8835e-004,<B>
+43-2484,CC_residual_temperature_correction_factor_c,-2.7001e-006,<C>
+43-2484,CC_residual_temperature_correction_factor_e,3.6000e-002,<E>
+43-2484,CC_voltage_offset,-0.4536,<offset> - TBD replaces frequency_offset
+43-2484,CC_frequency_offset,-0.4536,<offset> - Oxygen_SBE43_2484.xml
```

Comment:
Please note that this is deprecated by voltage_offset. (Will replace in the future.)